### PR TITLE
Added /poster page for MiL poster. Also added a layout for using Math…

### PR DESCRIPTION
**Pull request by feffemannen**
…Jax.

### CODE REVIEW

**Comment by greleigh** [comment]
Sorry for being slow to this! The layout ‘page’ (and default) already includes mathjax. Did you check if the page works using “layout: page”?

File: _pages/poster.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+The only two cases in which $A \leftrightarrow (B \leftrightarrow A)$ is the
+first and third row, where $B$ is true as well.
+
+Therefore, $B$ is true and **you should study logic**.
+
+## Paradoxes
+
+This statement is "self referential", but not paradoxical. However, it's easy
+to come with a self refential paradoxical statement:
```
Comment (greleigh) [inline _pages/poster.md:81]: “referential”

File: _pages/poster.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+---
+layout: poster
+title: You should study logic iff this sentence is true.
+---
+{% assign wiki = "https://en.wikipedia.org/wiki/Liar_paradox" %}
+{% assign mil = "https://www.gu.se/en/study-gothenburg/logic-masters-programme-h2log" %}
```
Comment (greleigh) [inline _pages/poster.md:6]: `{% assign mil = site.data.links['MiL'].url %}`

File: _pages/poster.md (hunk 1 above)
Comment (feffemannen) [reply _pages/poster.md:81]: Thanks.

File: _pages/poster.md (hunk 2 above)
Comment (feffemannen) [reply _pages/poster.md:6]: Great. Thanks.

File: _pages/poster.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
+---
+layout: poster
```
[the remark this answers is not in the document]
Comment (feffemannen) [reply _pages/poster.md:2]: Aha. But, it seems that inline code isn't working as I suspected.

* $a \rightarrow b$ is not being rendered
* $$a \rightarrow b$$ is being rendered as inline math.

That's way I missed that it was already included.

However, in mathjax_support.html is says
```
     inlineMath: [ ['$', '$'] ],
    displayMath: [ ['$$', '$$'] ],
```
which seems to indicate another behavior. Should I just use $$a \rightarrow b$$?

[the remark this answers is not in the document]
Comment (greleigh) [reply _pages/poster.md:2]: Ah. Yes, I changed the MathJax defaults so `$ ⋯ $` gives inline and `$$ ⋯ $$` gives display. That way math can be used in abstracts etc without much effort.
So, yes, use `$$ ⋯ $$` for displayed

[the remark this answers is not in the document]
Comment (greleigh) [reply _pages/poster.md:2]: Oh, hang on. I now understand the problem — our implementation is doing this wrong. I’ll take a look at it

[the remark this answers is not in the document]
Comment (greleigh) [reply _pages/poster.md:2]: This is how the page renders for me. Are you saying that the `$ ⋯ $` does not render inline, but as display?
[You should study logic iff this sentence is true. · Logic@GU.pdf](https://github.com/greleigh/Logic-GU/files/9817889/You.should.study.logic.iff.this.sentence.is.true.Logic%40GU.pdf)

[the remark this answers is not in the document]
Comment (feffemannen) [reply _pages/poster.md:2]: This is how it get rendered locally for me, when changing the layout of the page to "page":

image

Source:
image

[the remark this answers is not in the document]
Comment (feffemannen) [reply _pages/poster.md:2]: I.e., locally on my machine with layout "page" '$$ .. $$' gets rendered as inline and '$..$' isn't rendered.

**Comment by greleigh** [comment]
Thanks, I’ll have a closer look this evening /G

On 19 Oct 2022, at 13:21, feffemannen ***@***.***> wrote:

﻿

@feffemannen commented on this pull request.

________________________________

In _pages/poster.md<https://github.com/greleigh/Logic-GU/pull/67#discussion_r999307763>:

> @@ -0,0 +1,96 @@
+---
+layout: poster


I.e., locally on my machine with layout "page" '$$ .. $$' gets rendered as inline and '$..$' isn't rendered.

**Comment by greleigh** [comment]
OK, I’ve just “fixed” MathJax support on the main site. For some reason, all the config was in place but the script was never loaded! In previous use, I sometimes that the math code should be ‘protected’ in html tags so that Jekyll doesn’t process it. So if there is some stand-alone cases where the math doesn’t render you can also try `<span>$ f(a) = b^c $</span>`.


As I seem to be able to send commits to pull requests, I’ve added a commit here with the change to `layout: page` and where the html file should be stored in the web. If you can confirm it compiles for you correctly, then we send it off ;-)

**Comment by greleigh** [comment]
Ignore everything I’ve said so far! MathJax was working but I had forgotten how it works in markdown. Inline math blocks use `$$ … $$` since MathJax v3 and not `$ … $`. `\(` (inline) and `\[` (display) can also be used but they have to be protected from the markdown parser so they become `\\( a + b \\)` and `\\[ … \\]`.

**Comment by feffemannen** [comment]
Updated now. MathJax seems to behave a bit unpredictable. In one case '$$..$$' was displayed as displayed math if I didn't protect it within a '<span>'. 

Also, I'd like the url to be 'logic-gu.se/poster' if that's ok. It's nicer url as we want to print it on the poster.

**Comment by greleigh** [comment]
Great, it looks good to me and I’ll merge.

The markdown parser (kramdown) recognises math blocks and makes an educated guess at whether to mark it as inline or display. This gets overridden by the `span` tag (display mode can be forced using `<div>$$ ⋯ $$</div>`)

**Pull request merged**